gdal/ogr/ogrsf_frmts/elastic/ogrelasticdriver.cpp

```diff
@@ -105,7 +105,7 @@ void RegisterOGRElastic() {
     "<LayerCreationOptionList>"
     "  <Option name='INDEX_NAME' type='string' description='Name of the index to create (or reuse). By default the index name is the layer name.'/>"
     "  <Option name='INDEX_DEFINITION' type='string' description='Filename from which to read a user-defined index definition, or index definition as serialized JSon.'/>"
-    "  <Option name='MAPPING_NAME' type='string' description='Name of the mapping type within the index.' default='FeatureCollection'/>"
+    "  <Option name='MAPPING_NAME' type='string' description='Name of the mapping type within the index.' default='FeatureCollection'/>. This option is ignored when converting to Elasticsearch >=7 (see [Removal of mapping types](https://www.elastic.co/guide/en/elasticsearch/reference/current/removal-of-types.html)."
```

Comment:
This is invalid XML. The added comment should be moved to gdal/doc/source/drivers/vector/elasticsearch.rst

Reply:
Amending my comment: this is valid XML, but not how we proceed usually. So perhaps add at the description attribute " (ES < 7)", and the longer comment in the .rst